generic/tclcurl.h

```diff
@@ -488,13 +495,14 @@ void curlSetBodyVarName(Tcl_Interp *interp,struct curlObjData *curlDataPtr);
 char *curlstrdup (char *old);
 
 
-char *curlCreateShareObjCmd (Tcl_Interp *interp,struct shcurlObjData  *shcurlData);
+Tcl_Obj* curlCreateShareObjCmd (Tcl_Interp *interp,struct shcurlObjData  *shcurlData);
 int curlShareInitObjCmd (ClientData clientData, Tcl_Interp *interp,
         int objc,Tcl_Obj *CONST objv[]);
 int curlShareObjCmd (ClientData clientData, Tcl_Interp *interp,
         int objc,Tcl_Obj *CONST objv[]);
 int curlCleanUpShareCmd(ClientData clientData);
 
+#ifndef multi_h
```

Comment:
Is there any possibility that multi_h won't be set? multi.c (and thus multi.h) seem to be incorporated unconditionally in the config files.